vtr_flow/scripts/parse_vtr_task.pl

```diff
@@ -150,6 +150,7 @@ sub parse_single_task {
 	my $qor_parse_file;
         my $second_parse_file;
         my $counter = 0;
+	my $single_check_golden = $check_golden;
```

Comment:
It would be good to comment what this variable is (it's a complicated one).  

Should also have another comment should also probably explain the various ways this routine can operate 
 - (parse one result file for errors, parse a result and compare to a golden (prior) result, or parse two results files and make sure they are the same)?
 - I suspect the top of the file should also comment the various ways this parser can operate (may overlap with the function level comment -- use your judgement about where best to comment this, but we should comment that this parser can operate in different modes).

Reply:
@vaughnbetz  comments have been added and log files are now in the description above.

Reply:
Thanks!